webapp/security/alembic/versions/8e9386e7a94b_add_component_column_to_status_table.py

```diff
@@ -0,0 +1,35 @@
+"""add component column to status table
+
+Revision ID: 8e9386e7a94b
+Revises: fc282ca1dc4d
+Create Date: 2020-07-16 15:01:39.739465
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "8e9386e7a94b"
+down_revision = "fc282ca1dc4d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    components = ["main", "universe"]
+    components_str = "'" + "','".join(components) + "'"
```

Comment:
This seems unnecessarily complex - what's wrong with:

```suggestion
    components = "main, universe"
```
?

Reply:
I cleaned this up.